src/presenter/list-presenter.js

```diff
@@ -0,0 +1,61 @@
+import ListFilterView from '../view/list-filter-view.js';
+import ListSortView from '../view/list-sort-view.js';
+import ListContainerView from '../view/list-container-view.js';
+import EditPointView from '../view/edit-point-view.js';
+import ItemView from '../view/item-view.js';
+import {render} from '../render.js';
+import {getDefaultPoint, getDefaultDestination} from '../view/edit-point-view.js';
+
+const filtersContainer = document.querySelector('.trip-controls__filters');
+const sortContainer = document.querySelector('.trip-events');
+
+export default class ListPresenter {
+  listComponent = new ListContainerView();
+
+  constructor({existPointsModel}) {
+    this.existPointsModel = existPointsModel;
+  }
+
+  init() {
+    this.listPoints = [...this.existPointsModel.getPoints()];
+
+    render(new ListFilterView(), filtersContainer);
+    render(new ListSortView(), sortContainer);
+    render(this.listComponent, sortContainer);
+
+    render(new EditPointView({
+      points: getDefaultPoint(),
```

Comment:
Если речь идет о коллекции тогда уместен множественный нейминг если нет то пишем в единственном числе
```suggestion
      point: getDefaultPoint(),
```